src/category_theory/category/Pointed.lean

```diff
@@ -4,6 +4,9 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
 -/
 import category_theory.concrete_category.basic
+import category_theory.monoidal.category
+import category_theory.monoidal.of_chosen_finite_products
+import category_theory.limits.shapes.types
```

Comment:
Are both of these imports still needed? It doesn't look to me like you use either of them, but perhaps I'm missing something